benchmarks/span.js

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+var _ = require('lodash');
+var Benchmark = require('benchmark');
+var benchmarks = require('beautify-benchmark');
+var ConstSampler = require('../dist/src/samplers/const_sampler.js').default;
+var InMemoryReporter = require('../dist/src/reporters/in_memory_reporter.js').default;
+var SpanContext = require('../dist/src/span_context.js').default;
+var Tracer = require('../dist/src/tracer.js').default;
+var opentracing = require('opentracing');
+
+function benchmarkSpan() {
+    var tracer = new Tracer('benchmark-tracer', new InMemoryReporter(), new ConstSampler(true));
```

Comment:
calling ops on sampled and non-sampled span has different perf. If anything, we're probably more interested in the perf of un-sampled spans since they occur a lot more often,